suites/nautilus/cephfs/tier-2_cephfs_test-quota.yaml

```diff
@@ -0,0 +1,160 @@
+#                     Set max bytes quota to a number(say 1Gb) and fill data until it reaches the limit and verify if the set
+#                     quota limit is working fine. Remove the quota once it reaches the max bytes and try adding more
+#                     data, verify if set quota is removed. Repeat the procedure for few more times.
+#   CEPH-83573408     Test to validate the quota remains intact even after rebooting the Node.
+#                     Create a FS and create 10 directories and mount them on kernel client and fuse client(5 mounts
+#                     each). Set max bytes quota to a number(say 1Gb) and also set max files quota (say 20) and verify if
+#                     the set quota limit is working fine by filling max number of files and also by filling data to reach
+#                     the max limit. Reboot the node , once the node is up verify if the set quota remains or not.
+#   CEPH-83573406     Test to validate the combination of quota_max_bytes and quota_max_files
+#                     Create a FS and create 10 directories and mount them on kernel client and fuse client(5 mounts each)
+#                     Set max bytes quota to a number(say 1Gb) and also set max files quota (say 20) and verify if the set
+#                     quota limit is working fine by filling max number of files and also by filling data to reach the max
+#                     limit. Increase the limit to higher number and add data and files again. Repeat this for few more
+#                     times.
+#===========================================================================================================================
+tests:
+  - test:
+      name: install ceph pre-requisites
+      module: install_prereq.py
+      abort-on-fail: true
+
+  - test:
+     name: ceph ansible
+     module: test_ansible.py
+     config:
+       ansi_config:
+           ceph_test: True
+           ceph_origin: distro
+           ceph_repository: rhcs
+           osd_scenario: lvm
+           osd_auto_discovery: False
+           journal_size: 1024
+           ceph_stable: True
+           ceph_stable_rh_storage: True
+           fetch_directory: ~/fetch
+           copy_admin_key: true
+           dashboard_enabled: False
+           ceph_conf_overrides:
+               global:
+                 osd_pool_default_pg_num: 64
+                 osd_default_pool_size: 2
+                 osd_pool_default_pgp_num: 64
+                 mon_max_pg_per_osd: 1024
+               mon:
+                 mon_allow_pool_delete: true
+                 debug mon: 5
+               mds:
+                 mds_bal_split_size: 100
+                 mds_bal_merge_size: 5
+                 mds_bal_fragment_size_max: 10000
+                 debug mds: 5
+           cephfs_pools:
+             - name: "cephfs_data"
+               pgs: "8"
+             - name: "cephfs_metadata"
+               pgs: "8"
```

Comment:
These settings would put the cluster in a warn state... please revisit the pg values